src/pages/iou/steps/IOUParticipantsPage/IOUParticipantsRequest.js

```diff
@@ -0,0 +1,137 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import {hide as hideSidebar} from '../../../../libs/actions/Sidebar';
+import withWindowDimensions, {
+    windowDimensionsPropTypes,
+} from '../../../../components/withWindowDimensions';
+import {getNewChatOptions} from '../../../../libs/OptionsListUtils';
+
+import OptionsSelector from '../../../../components/OptionsSelector';
+import ONYXKEYS from '../../../../ONYXKEYS';
+
+const personalDetailsPropTypes = PropTypes.shape({
+    // The login of the person (either email or phone number)
+    login: PropTypes.string.isRequired,
+
+    // The URL of the person's avatar (there should already be a default avatar if
+    // the person doesn't have their own avatar uploaded yet)
+    avatar: PropTypes.string.isRequired,
+
+    // This is either the user's full name, or their login if full name is an empty string
+    displayName: PropTypes.string.isRequired,
+});
+
+const propTypes = {
+    // Callback to inform parent modal of success
+    onStepComplete: PropTypes.func.isRequired,
+
+
+    // All of the personal details for everyone
+    personalDetails: PropTypes.objectOf(personalDetailsPropTypes).isRequired,
```

Comment:
Hmmm this is a bit confusing, did you mean `PropTypes.arrayOf(personalDetailsPropTypes)`?

Reply:
@roryabraham  Same with this! The component gets the prop of type object from Onyx if I am not mistaken. 

This is the console log of the prop. 

![Screenshot 2021-03-16 at 08 08 51](https://user-images.githubusercontent.com/19339044/111246769-f3183680-862e-11eb-81f0-3f5a1202201c.png)

The getNewGroupOptions function maps it into an array for us to work with.

Please let me know if you have any suggestions for me to make it less confusing.

Reply:
I think this is okay, as it matches our existing usages.